tasks/mutations/ward_mutations.ts

```diff
@@ -119,28 +119,26 @@ export const useWardDetailsQuery = (wardId?: string, organisationId?: string) =>
   })
 }
 
-export const useWardQuery = (id: string, organisationId?: string) => {
-  return useQuery({
-    queryKey: [wardsQueryKey, id],
-    enabled: !!id,
-    queryFn: async (): Promise<WardWithOrganizationIdDTO> => {
-      const req = new GetWardRequest()
-      req.setId(id)
-      const res = await wardService.getWard(req, getAuthenticatedGrpcMetadata(organisationId))
-
-      if (!res.toObject()) {
-        console.error('error in Ward query')
-      }
+export const useWardQuery = (id: string, organisationId?: string) => useQuery({
+  queryKey: [wardsQueryKey, id],
+  enabled: !!id,
+  queryFn: async (): Promise<WardWithOrganizationIdDTO> => {
+    const req = new GetWardRequest()
+    req.setId(id)
+    const res = await wardService.getWard(req, getAuthenticatedGrpcMetadata(organisationId))
 
-      const ward: WardWithOrganizationIdDTO = {
-        id: res.getId(),
-        name: res.getName(),
-        organizationId: res.getOrganizationId()
-      }
-      return ward
+    if (!res.toObject()) {
+      console.error('error in Ward query')
```

Comment:
throw error? Haven't you applied that to every `console.error` pattern?

Reply:
I wanted to refactor this hook a bit more and haven't gotten around to that yet; didn't want to just partially update the code